diagnostic.md

```diff
@@ -50,21 +50,16 @@ Using Atom, open up `diagnostic.md` and write your answers below (where indicate
 
 1. Is `/Users/blah_blah/Desktop` a relative path or an absolute path? How do you know?
 
- <!-- Answer Starts Here -->
-
- <!-- Answer Ends Here -->
+ This is a root path because it includes the root directory(user)
 
  2. Suppose that we're working on a project, and we want to use a font that's being hosted on Google at `https://fonts.googleapis.com/css?family=Open+Sans`. Is that an absolute or relative path to refer to it? Why?
 
- <!-- Answer Starts Here -->
+It is an absolute path, because it is a link to a website and is its own path separate from my system.
```

Comment:
This is an absolute path since its location is the same no matter where it's being referred from.